src/main/java/school/lemon/changerequest/java/DemoMatrix.java

```diff
@@ -0,0 +1,30 @@
+package school.lemon.changerequest.java;
+
+/**
+ * Created by lera on 12.12.16.
+ */
+public class DemoMatrix {
+    public static void main(String[] args) {
+
+        double[][] matrix_A = {{4., 2.}, {9., 0.}};
```

Comment:
Try use more readable formats when you're declaring variables, for 2d array it'll be:
```java
double[][] matrixA = {
           {4., 2.},
           {9., 0.}
};
```